Formula/protozero.rb

```diff
@@ -0,0 +1,30 @@
+class Protozero < Formula
+  desc "Minimalistic protocol buffer decoder and encoder in C++"
+  homepage "https://github.com/mapbox/protozero"
+  url "https://github.com/mapbox/protozero/archive/v1.3.0.tar.gz"
+  sha256 "85f9238fa662ff06a1e364f1461846a9d377846274e7f98407307e31086cab2b"
+
+  def install
+    (include/"protozero").install Dir["include/protozero/*"]
+  end
+
+  test do
+    files = ["byteswap.hpp", "config.hpp", "exception.hpp",
+             "pbf_builder.hpp", "pbf_message.hpp", "pbf_reader.hpp",
+             "pbf_writer.hpp", "types.hpp", "varint.hpp", "version.hpp"]
+    files.each do |f|
+      assert_equal(File.exist?("#{include}/protozero/#{f}"), true)
+    end
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <protozero/version.hpp>
+      int main() {
+        if (PROTOZERO_VERSION_STRING == "#{version}") {
+          return 0;
+        }
+        return 1;
+      }
+    EOS
+    system "c++", testpath/"test.cpp", "-o", "test"
```

Comment:
Please use `ENV.cxx` instead of `"c++"` and please add another argument `"-I#{include}"` to the invocation, so that the test still works even if Homebrew is not installed to `/usr/local`. You can also use `"test.cpp"` instead of `testpath/"test.cpp"` as `testpath` is the working directory when the test executes. To summarize:

```ruby
system ENV.cxx, "-I#{include}", "test.cpp", "-o", "test"
```